src/selectedtests/app/app.py

```diff
@@ -37,6 +38,12 @@ def create_app(mongo_wrapper: MongoWrapper, evg_api: EvergreenApi) -> FastAPI:
         prefix="/projects/{project}/test-mappings",
         tags=["projects"],
     )
-    app.state.db = mongo_wrapper
-    app.state.evg_api = evg_api
+
+    def dependencies(binder: inject.Binder):
+        binder.bind(EvergreenApi, get_evg_api())
+        binder.bind('EvergreenApi', get_evg_api())
+        binder.bind(MongoWrapper, get_mongo_wrapper())
+        binder.bind('MongoWrapper', get_mongo_wrapper())
```

Comment:
These string bindings are needed since `from __future__ import annotations` makes all of our type annotations strings instead --> `inject` breaks when looking for these strings since we only registered classes. Will talk with `inject` maintainers if it makes sense to register these automatically if omitted since otherwise their package doesn't mesh well with any classes that reference themselves.